src/widget.cpp

```diff
@@ -2253,15 +2253,17 @@ void NWidgetBackground::SetupSmallestSize(Window *w)
 		if (w == nullptr) return;
 
 		if (this->type == WWT_FRAME) {
+			std::string text = GetStringForWidget(w, this);
+			Dimension text_size = text.empty() ? Dimension{0, 0} : GetStringBoundingBox(text, this->text_size);
```

Comment:
Wouldn't it be better to have `GetStringBoundingBox` always return `{0,0}` if an empty string is provided? Any other result seems weird to me.

Reply:
I'd rather not touch something that might have unexpected impact just for this one case.

Reply:
Fair enough, I've had my fair share of those "quick fixes" myself hehe